alpenhorn/auto_import.py

```diff
@@ -225,14 +312,32 @@ def add_acq(acq_type, name, node, comment=""):
     if ac.ArchiveAcq.select(ac.ArchiveAcq.id).where(ac.ArchiveAcq.name == name).count():
         raise AlreadyExists('Acquisition "%s" already exists in DB.' % name)
 
+    # Extended acq class
+    ExtendedArchiveAcq = extensions.extended_table(ac.ArchiveAcq)
+
+    # Get the extended table data, if any
+    ext_data = _get_extended_table_data(
+        ExtendedArchiveAcq, acq_name, acq_type, file_name, None, node, path
```

Comment:
The `None` here should be the `FileType`, but that doesn't exist at this point because this call is happening too early in this version of the code.

A later PR will fix this.